community/modules/compute/gke-node-pool/README.md

```diff
@@ -28,6 +28,91 @@ can be overridden using the `taints` setting. See
 [docs](https://cloud.google.com/kubernetes-engine/docs/how-to/node-taints) for
 more info.
 
+### Considerations with GPUs
+
+When a GPU is attached to a node an additinal taint is automatically added:
+`nvidia.com/gpu=present:NoSchedule`. For jobs to get placed on these nodes the
+equivalent toleration is required. When using the `gke-job-template` module this
```

Comment:
"The `gke-job-template` module will automatically apply..."

Reply:
Done.